packages/flutter/test/widgets/inherited_dependencies.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
One more: shouldn't the file name end in `_test.dart`?

Reply:
Uh D'Oh